articles/active-directory/develop/quickstart-register-app.md

```diff
@@ -39,7 +39,8 @@ Follow these steps to create the app registration:
 1. If you have access to multiple tenants, in the top menu, use the **Directory + subscription** filter :::image type="icon" source="./media/common/portal-directory-subscription-filter.png" border="false"::: to select the tenant in which you want to register an application.
 1. Search for and select **Azure Active Directory**.
 1. Under **Manage**, select **App registrations** > **New registration**.
-1. Enter a **Name** for your application. Users of your app might see this name. You can change it later.
+1. Enter a **display Name** for your application. Users of your application might see the display name when they use the app, for example during sign-in.
+You can change the display name at any time and multiple app registrations can share the same name. The app registration's automatically generated Application (client) ID, not its display name, uniquely identifies your app within the identity platform.
```

Comment:
```suggestion
1. Enter a display **Name** for your application. Users of your application might see the display name when they use the app, for example during sign-in.
    You can change the display name at any time and multiple app registrations can share the same name. The app registration's automatically generated Application (client) ID, not its display name, uniquely identifies your app within the identity platform.
```

* Bolds only the UI label of the element that the user is to modify (adheres to convention)
* Indents second line four spaces so it doesn't break the numbering.